### PR TITLE
implemented #398

### PR DESCRIPTION
**Pull request by ShamaUgale**
implemented #398
1. changed the schema files to accommodate machineIPs as a list

### CODE REVIEW

File: caps/capabilities.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   },
   "hostMachines": [
     {
-      "machineIP": "127.0.0.1",
+      "machineIP": ["127.0.0.1"],
```
Comment (SrinivasanTarget) [inline caps/capabilities.json]: @ShamaUgale What if there are different xcode versions with different machines. So we should ideally accept both formats string and array.

Case 1: Diff sim's or diff xcode versions on diff machines
Case 2: lot of machines (ips) with same sim and xcode versions.

**Comment by saikrishna321** [comment]
@ShamaUgale Thanks for the PR. Can you check whats happening in this line 

https://github.com/saikrishna321/AppiumTestDistribution/blob/master/src/main/java/com/appium/manager/LocalAppiumManager.java#L144-L146

File: caps/capabilities.json (hunk 1 above)
Comment (ShamaUgale) [reply caps/capabilities.json]: @SrinivasanTarget wrt to "case 1", are you also referring to OS being a list/string?

Comment (saikrishna321) [reply caps/capabilities.json]: So this PR covers Case2. But will break for Case 1. As srini mentioned we need to support both String and Array for machineIP

Comment (ShamaUgale) [reply caps/capabilities.json]: i am accommodating it.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -8,8 +8,8 @@ pool:
 
 steps:
 - script: |
-	export APPIUM_BINARY_PATH="/usr/local/lib/node_modules/appium/build/lib/main.js"
-	export NODE_BINARY_PATH="/usr/local/opt/node@8/bin/node"
+    export APPIUM_BINARY_PATH="/usr/local/lib/node_modules/appium/build/lib/main.js"
+    export NODE_BINARY_PATH="/usr/local/opt/node@8/bin/node"
```
Comment (SrinivasanTarget) [inline azure-pipelines.yml]: @ShamaUgale Are you the mac images we use above has node 8 by default?

Comment (saikrishna321) [reply azure-pipelines.yml]: @ShamaUgale you need to revert this as we are not sure if your fix would work, as your overriding user given path of server with APPIUM_BINARY_PATH

**Pull request merged**